tools/shared.py

```diff
@@ -1212,7 +1212,6 @@ def __init__(self):
 
     @classmethod
     def reset(self):
-      self.attrs = {'QUANTUM_SIZE': 4}
```

Comment:
I agree we don't need the quantum stuff here (that's the default value anyhow), but I think we do need `self.attrs = {}` here, so we have a blank slate before doing that load?

Reply:
@kripken 
done